specs/disciplina/witness/api/explorer.yaml

```diff
@@ -367,6 +367,7 @@ definitions:
             * `InvalidFormat` - Failed to deserialise one of parameters.
 
             * `BlockNotFound` - Specified block does not exist.
+            * `NotEnoughMoney` - Transaction's source account has not enough money to make this transaction.
```

Comment:
We should probably add tx submitting endpoint to the docs too, at this point

Reply:
Oh, it's not there after all.
Would it be fair to add such endpoint to `explorer.yaml`? :thinking:

Reply:
Actually, there's no particular reason to consider "block explorer API" some special part of witness API, so this should probably be renamed to `witness.yaml` or something